js/lib/k6-shim.js

```diff
@@ -0,0 +1,198 @@
+//require('./modules/es6.date.to-iso-string');
+//require('./modules/es6.date.to-string');
+//require('./modules/es6.date.to-primitive');
+//require('./modules/es6.array.is-array');
+//require('./modules/es6.array.from');
+//require('./modules/es6.array.of');
+//require('./modules/es6.array.join');
+//require('./modules/es6.array.slice');
+//require('./modules/es6.array.sort');
+//require('./modules/es6.array.for-each');
+//require('./modules/es6.array.map');
+//require('./modules/es6.array.filter');
+//require('./modules/es6.array.some');
+//require('./modules/es6.array.every');
+//require('./modules/es6.array.reduce');
+//require('./modules/es6.array.reduce-right');
+//require('./modules/es6.array.index-of');
+//require('./modules/es6.array.last-index-of');
+//require('./modules/es6.array.copy-within');
+//require('./modules/es6.array.fill');
+//require('./modules/es6.array.find');
+//require('./modules/es6.array.find-index');
+//require('./modules/es6.array.species');
+//require('./modules/es6.array.iterator');
+//require('./modules/es6.regexp.constructor');
+//require('./modules/es6.regexp.to-string');
+//require('./modules/es6.regexp.flags');
+//require('./modules/es6.regexp.match');
+//require('./modules/es6.regexp.replace');
+//require('./modules/es6.regexp.search');
+//require('./modules/es6.regexp.split');
+//require('./modules/es6.promise');
+//require('./modules/es6.map');
+//require('./modules/es6.set');
+//require('./modules/es6.weak-map');
+//require('./modules/es6.weak-set');
+//require('./modules/es6.typed.array-buffer');
+//require('./modules/es6.typed.data-view');
+//require('./modules/es6.typed.int8-array');
+//require('./modules/es6.typed.uint8-array');
+//require('./modules/es6.typed.uint8-clamped-array');
+//require('./modules/es6.typed.int16-array');
+//require('./modules/es6.typed.uint16-array');
+//require('./modules/es6.typed.int32-array');
+//require('./modules/es6.typed.uint32-array');
+//require('./modules/es6.typed.float32-array');
+//require('./modules/es6.typed.float64-array');
+//require('./modules/es6.reflect.apply');
+//require('./modules/es6.reflect.construct');
+//require('./modules/es6.reflect.define-property');
+//require('./modules/es6.reflect.delete-property');
+//require('./modules/es6.reflect.enumerate');
+//require('./modules/es6.reflect.get');
+//require('./modules/es6.reflect.get-own-property-descriptor');
+//require('./modules/es6.reflect.get-prototype-of');
+//require('./modules/es6.reflect.has');
+//require('./modules/es6.reflect.is-extensible');
+//require('./modules/es6.reflect.own-keys');
+//require('./modules/es6.reflect.prevent-extensions');
+//require('./modules/es6.reflect.set');
+//require('./modules/es6.reflect.set-prototype-of');
+//require('./modules/es7.array.includes'); // in goja
+require('./modules/es7.array.flat-map');
+// this is now called flat, so maybe drop it https://github.com/tc39/proposal-flatMap 
+require('./modules/es7.array.flatten');
```

Comment:
We should support `Array.prototype.flat` though, although I think the use case might be somewhat limited